CMakeLists.txt

```diff
@@ -7,7 +7,16 @@ set(Required_Gpgmepp_Version 1.13.1)
 
 cmake_minimum_required(VERSION ${Required_CMake_Version})
 
-PROJECT(ledger)
+if (NOT SKBUILD)
+  set(SKBUILD_PROJECT_NAME ledger)
+  set(SKBUILD_PROJECT_VERSION 3.3.2)
+endif()
+
+project(
+  ${SKBUILD_PROJECT_NAME}
+  VERSION ${SKBUILD_PROJECT_VERSION}
+  LANGUAGES C CXX
+  )
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/doc")
 include(LedgerVersion)
```

Comment:
Handling of version numbers could use a clean-up, I'll leave this for future me.